src/azure-cli/azure/cli/command_modules/appconfig/commands.py

```diff
@@ -64,6 +70,12 @@ def get_custom_sdk(custom_module, client_factory, table_transformer):
         g.show_command('show', 'show_configstore')
         g.show_command('show-deleted', 'show_deleted_configstore', is_preview=True, table_transformer=deleted_configstore_output_format)
 
+    with self.command_group('appconfig replica', configstore_replica_util, is_preview=True) as g:
+        g.command('list', 'list_replica', table_transformer=configstore_replica_output_format)
```

Comment:
> table_transformer=configstore_replica_output_format [](http://example.com/codeflow?start=41&length=52)

Since ```configstore_replica_output_format``` is specified in the command group level in ```configstore_replica_util```, do we still need to add per command?

Reply:
You are right, we don't need to specify the table formatter for individual commands anymore.
Removed it.